Machine Learning/ProblemChild/datafeeds/datafeed-experimental-rare-process--by-parent-problemchild.txt

```diff
@@ -0,0 +1,54 @@
+PUT _ml/datafeeds/experimental-rare-process-by-parent-problemchild
```

Comment:
Remove `PUT` statements from `datafeed` files.

change file extension from `txt` to `json`